src/views/empty-state.component.js

```diff
@@ -117,6 +117,10 @@ angular.module('patternfly.views').component('pfEmptyState', {
       if (angular.isUndefined(ctrl.config)) {
         ctrl.config = {};
       }
+      if (ctrl.config.helpLink && angular.isUndefined(ctrl.config.helpLink.url)) {
+        // if no url specified, set url to not redirect.  ie. just do urlAction
+        ctrl.config.helpLink.url = "javascript:void(0)";
+      }
       ctrl.updateConfig();
```

Comment:
This needs to be done in $onChanges as well for the case where the helpLink was added to the config object after init. Maybe do this as part of updateConfig.

Reply:
Fixed, added a `$doCheck` since we need to do a deep compare on the config obj.